frontend/src/components/prompts/Rename.vue

```diff
@@ -92,7 +92,7 @@ export default {
       try {
         await api.move([{ from: oldLink, to: newLink }]);
         if (!this.isListing) {
-          this.$router.push({ path: newLink });
+          this.$router.replace({ path: newLink });
```

Comment:
It's a fix for the “back” button function. If we use push. the route link with the old name still in the route history. and when we back to it, an error will show. So we need to change it to replace.